src/test/java/teammates/logic/core/FeedbackResponseCommentsLogicTest.java

```diff
@@ -31,6 +58,7 @@ protected void prepareTestData() {
     @BeforeMethod
     public void refreshTestData() {
         dataBundle = getTypicalDataBundle();
+        courseRosterDataBundle = loadDataBundle("/CourseRosterDataBundle.json");
```

Comment:
Are you sure that the typical data bundle is insufficient to test this function?

Reply:
I would like to have a feedback response of student from one team to student of another team. typical data bundle have one such response which is between student 2 in course 1 and student 5 in course 1. However, student 2 in course 1 is the same student as student 2 in course 2. When the data bundle is loaded, this student will be treated as student 2 in course 2 instead of student 2 in course 1 and the team that this student is in is the team in course 2. It creates some ambiguityies here.

Reply:
I am not sure what you mean by the above. One student can be in multiple courses. How about you paste to me which cases the above affects and I try to rewrite it purely using the typical bundle?

Reply:
I have created a new test case to illustrate my point. It is from line 471 to 478.

Reply:
Upon further thought, I guess it is not a huge harm creating this bundle here.

Even if it seems possible to refactor this to use purely the typical data bundle, it would require some re-engineering of the test. It may not be worth the effort required.